.github/workflows/trigger-e2e-tests.yml

```diff
@@ -0,0 +1,108 @@
+name: Trigger E2E Tests
+
+on:
+  pull_request:
+    types:
+      - ready_for_review
+  workflow_call:
+    inputs:
+      build-tag:
+        required: true
+        type: string
+
+concurrency:
+  # Allow only one workflow per any non-`main` branch.
+  group: ${{ github.workflow }}-${{ github.ref_name }}-${{ github.ref_name == 'main' && github.sha || 'anysha' }}
+  cancel-in-progress: false
```

Comment:
It might be ok to drop the `concurrency` block completely for this workflow. What do you think?
It should fix GitHub's deadlocks. Are there any drawbacks?

Reply:
make sense just to make things easy.

Reply:
done